addons/mail/data/mail_templates_chatter.xml

```diff
@@ -46,6 +46,24 @@
 </div>
         </template>
 
+        <template id="mail.message_document_unfollowed">
```

Comment:
Shouldn't it be in "mail_templats_public.xml" ? This files is more for qweb templates used to log messages in chatter :)

Reply:
Yes you are right, it is not for rendering in the mail but in the browser.